src/main/java/games/strategy/engine/data/GameParser.java

```diff
@@ -926,14 +942,15 @@ private void parseProperties(final Node root) throws GameParseException {
       data.getProperties()
           .addEditableProperty(new NumberProperty(Constants.AI_BONUS_DEFENSE, null, data.getDiceSides(), 0, 0));
     }
+    properties.set(Constants.MAP_NAME, mapCodeName);    // Overwrite map name with containing folder
```

Comment:
Could you also provide some pointers on how we are consuming this property @simon33-2 ? I'm interested if we can make that relationship more direct.